plinth/modules/help/templates/help_index.html

```diff
@@ -52,7 +52,8 @@
   <p>
     {% blocktrans trimmed %}
       Many FreedomBox contributors and users are also available on the
-      #freedombox channel of the irc.oftc.net IRC network.
+      <a href="https://webchat.oftc.net/?randomnick=1&channels=freedombox&prompt=1&uio=MT11bmRlZmluZWQb1">
+      #freedombox</a> channel of the irc.oftc.net IRC network.
     {% endblocktrans %}
```

Comment:
`uio=` part can be removed since it corresponds to default options anyway.  Also, it would be more inviting for the user to click the link if we write an explicit message that this link will open a web based chat.  Perhaps like:

```
"Join and request help on the <a>#freedombox</a> channel using the IRC web interface."
```